WordPress/src/main/java/org/wordpress/android/viewmodel/domainregister/DomainSuggestionsViewModel.kt

```diff
@@ -26,6 +29,12 @@ class DomainSuggestionsViewModel @Inject constructor(
 ) : ViewModel() {
     lateinit var site: SiteModel
     private var isStarted = false
+    private var debouncer = Debouncer()
+
+    @VisibleForTesting
```

Comment:
We try not to use `@VisibleForTesting` annotation unless it's not possible to test the code without it. We also prefer using Dependency injection over manually instantiation. I believe we can easily make Debouncer injectable and inject it using Dagger in production and manually inject a mock in tests. Wdyt?